precompiles/ArbOwner.go

```diff
@@ -497,11 +498,6 @@ func (con ArbOwner) SetMultiGasPricingConstraints(
 	evm mech,
 	constraints []MultiGasConstraint,
 ) error {
-	limit := l2pricing.MultiGasConstraintsMaxNum
```

Comment:
better to make this depend on arbosVersion then remove entirely (I think this check won't spend gas so won't be a problem to add)